packages/core/admin/admin/src/pages/MarketplacePage/components/EmptyPluginSearch/EmptyPluginGrid.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box } from '@strapi/design-system/Box';
+import { GridLayout } from '@strapi/design-system/Layout';
+
+const EmptyPluginCard = styled(Box)`
+  background: linear-gradient(180deg, rgba(234, 234, 239, 0) 0%, #eaeaef 100%);
```

Comment:
Can you use the colour from the design system? `#eaeaef`  is called `neutral150` ?